apps/accounts/throttles.py

```diff
@@ -0,0 +1,27 @@
+from rest_framework.throttling import SimpleRateThrottle
+
+from django.core.cache import caches
+
+from django.conf import settings
+
+
+class ResendEmailThrottle(SimpleRateThrottle):
+    """
+    Used to limit the requests to /accounts/user/resend-email
```

Comment:
can you also fix the endpoint to `/accounts/user/resend_email_verification`

Reply:
Done.